update_presenze.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/python3
```

Comment:
Sarebbe meglio: `#!/usr/bin/env python3`

Motivo: non su tutti i sistemi (e.g. macOS) python3 è detto sia presente in `/usr/bin/python3`, oppure è presente ma è una versione obsoleta, oppure l'utente potrebbe aver aggiunto al suo PATH una versione di python3 che sovrascrive quello di sistema.